### DISCUSSION TITLE
Is it possible having multiple models into a single AR?

### DISCUSSION THREAD

**Discussion by rksambari** [google/model-viewer#3737] [General]
Hi,

Using model viewer, can we have multiple models appear on AR mode and be able to move each model individually?
Can we have 2 model viewer HTML tags inside our single HTML file?

Thanks.

**Comment by btutal** [comment]
Hi, I am definitely no expert but to have multiple models on AR mode, you should set your model/scene that way. (Go with WebXR way etc.)  For iOS, you can serve `.reality` scenes with multiple models and interactions but they wouldn't come from two different model viewer tags in one HTML file. 

On the other hand, having more than one model-viewer element inside a single HTML is definitely possible. Complete model-viewer documentation works similar way :) Just check [this page ](https://modelviewer.dev/examples/loading/)

**Reply by elalish** [reply]
@btutal is correct; we have some thoughts about independently moving models around in AR, but it hasn't been implemented yet.

**Reply by rksambari** [reply]
@elalish Thanks for the response,

I just wanted to check if there are any timelines on this, that way it becomes easier for us to plan.

**Reply by elalish** [reply]
Well, I'm hoping to get multiple-model support in the next few months, but it'll only be programmatic movement. If you're hoping for a nice AR-UX for dragging things around, I could honestly use some UX mocks. Are you interested in collaborating?

**Reply by domingohernani** [reply]
Hello everyone! Are there any updates on this feature? My team is currently developing a capstone project at our university, and our panelist asked us if we could implement multiple models into a single AR

**Reply by elalish** [reply]
It's in progress, but not sure of timeline.
